packages/contracts/Releases.md

```diff
@@ -14,7 +14,7 @@ DAO_ENSSubdomainRegistrar_Implementation: 0x92ABB80D541c0Ea5C4C0CeB8c8EAA7c26447
 DAO_ENSSubdomainRegistrar_Proxy: 0x99C7D164fcd6a6F11E0C11fC7e932D285b7F08aB
 DAO_Implementation: 0xDA18018D87836394bae8e48beF910f7984ff7333
 DAO_Proxy: 0xe0C6B6f2078bbC38Bf71af47C286766689D3BaaF
-ERC20VotingSetup: 0xE75B0C19495CbA675e45Cb662D60429dee4de344
+TokenVotingSetup: 0xE75B0C19495CbA675e45Cb662D60429dee4de344
```

Comment:
Please don't rename it here. old contracts are deployed with a different name

Reply:
Fixed it